query/math.go

```diff
@@ -49,6 +49,12 @@ func (v *MathValuer) Call(name string, args []influxql.Expr) (interface{}, bool)
 			return v.callTrigFunction(math.Cos, args[0])
 		case "tan":
 			return v.callTrigFunction(math.Tan, args[0])
+		case "floor":
+			return v.callTrigFunction(math.Floor, args[0])
```

Comment:
I think you'll need to implement a new function for this. The argument type for these three functions should probably be restricted to only floats. It doesn't make sense to call any of these with an integer. I think these functions should also return integers rather than floats.

Reply:
callRoundFunction will called now.